glide.yaml

```diff
@@ -46,7 +46,6 @@ import:
   - lib/upgrade
 - package: github.com/termie/go-shutil
 - package: google.golang.org/grpc
-  version: 8050b9cbc271307e5a716a9d782803d09b0d6f2d
```

Comment:
pinning to a particular version seems more robust than floating against `master`. Is there a specific release we could use for `version`?

Reply:
No - the model we have is that we don't pin versions in glide.yaml unless we have a specific need to, they are pinned in glide.lock.